js/enemy.js

```diff
@@ -1,8 +1,8 @@
-// Enemy model
+var DEFAULT_ENEMY_SIZE = 10;
```

Comment:
Good. But now that we're polluting the global name spaces with variables, we should probably wrap this class (and the others) in an IIFE.